my-webxr-app/src/components/DataPoint.tsx

```diff
@@ -53,26 +52,41 @@ export default function DataPoint({
           : setSelectedDataPoint(id);
       }}
     >
+
+      {/* Spreading these props is arguably justified since it allows us to accept any number of
+       the very long list of possible mesh props. Without this, we would have to manually list
+       which ones we would and wouldn't accept */}
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
       <mesh {...meshProps}>
-        {/* Low numbers to try to minimize the number of faces we need to render*/}
+        {/* Low numbers to try to minimize the number of faces we need to render */}
         {/* There will be a LOT of these present in the simulation */}
-        <sphereGeometry args={size || [0.1, 10, 10]} />
+        <sphereGeometry args={size} />
         <meshStandardMaterial />
       </mesh>
 
       {/* This second mesh is the outline which works by rendering */}
       {/* only the BackSide of the mesh material */}
       <mesh
+        /* eslint-disable-next-line react/jsx-props-no-spreading */
         {...meshProps}
-        scale={outlineScale || 1.25}
-        visible={hoverCount != 0 || selectedDataPoint === id}
+        scale={outlineScale}
+        visible={hoverCount !== 0 || selectedDataPoint === id}
       >
-        <sphereGeometry args={size || [0.1, 10, 10]} />
+        <sphereGeometry args={size} />
         <meshStandardMaterial
-          color={selectedDataPoint === id ? "blue" : "aqua"}
+          color={selectedDataPoint === id ? 'blue' : 'aqua'}
           side={BackSide}
         />
       </mesh>
     </Interactive>
   );
 }
+
+/**
+ * Specify default values for DataPoint's optional props.
+ */
+DataPoint.defaultProps = {
+  outlineScale: 1.25,
+  size: [0.1, 10, 10],
+  meshProps: {},
+};
```

Comment:
ESLint required default values to be provided for the optional type members, so I refactored to permit that.